src/command/client/search/interactive.rs

```diff
@@ -229,15 +237,17 @@ impl State {
 
     #[allow(clippy::cast_possible_truncation)]
     fn draw_compact<T: Backend>(&mut self, f: &mut Frame<'_, T>, results: &[History]) {
+        let show_help = f.size().height > 1;
         let chunks = Layout::default()
             .direction(Direction::Vertical)
             .margin(0)
             .horizontal_margin(1)
             .constraints(
                 [
-                    Constraint::Length(1),
+                    Constraint::Length(u16::from(show_help)),
                     Constraint::Min(1),
                     Constraint::Length(1),
+                    Constraint::Length(2),
```

Comment:
In the final version, we may want to make this configurable:
```suggestion
                    Constraint::Length(2),
```

I did not go with a dynamic height to avoid having the input position change depending on the selected result command length.

Reply:
I was just about to ask about this haha, I think it is acceptable to just have it like this regardless 🤔

Reply:
Maybe this could be based on the longest command in history according to the current search and filter modes, ignoring the query. With a higher bound to avoid edge cases.

Reply:
And hidden by default if no line is longer than the widget width.

Reply:
Turns out it is not that perturbing to have a dynamic height based on the results even taking the query into account.